test/astar.cc

```diff
@@ -252,23 +271,36 @@ void assert_is_trivial_path(valhalla::Location& origin,
 
   Options options;
   create_costing_options(options);
```

Comment:
im not sure if you'd prefer it but you can use the functions in `valhalla/worker.h` to create an `Api` object which contains a properly constructed `Options` (and costing options) object from a json request.

Reply:
I mostly copied the code from some other test. Seems to work this way so currently inclined to leave it to rest.